### PR TITLE
Add GuestOrExternal user docs to v1.0

### PR DESCRIPTION
**Pull request by dmytrii-ilchuk**
This PR is tracked via work item 69119 in the Skilling ADO.

### CODE REVIEW

File: api-reference/v1.0/resources/conditionalaccessenumeratedexternaltenants.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# conditionalAccessEnumeratedExternalTenants resource type
+
+Namespace: microsoft.graph
+
+Represents a list of external tenants in the policy scope.
+
+## Properties
+
+| Property     | Type        | Description |
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/conditionalaccessenumeratedexternaltenants.md:20]: According to the schema, this resource inherits from the conditionalAccessExternalTenants abstract type. I've updated this topic and the corresponding beta version to inherit from conditionalAccessExternalTenants.

image

File: api-reference/v1.0/resources/conditionalaccessguestsorexternalusers.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# conditionalAccessGuestsOrExternalUsers resource type
+
+Namespace: microsoft.graph
+
+Represents internal guests and/or external users in the policy scope.
+
+## Properties
+
+| Property     | Type        | Description |
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/conditionalaccessguestsorexternalusers.md:18]: Is there a reason why the member `none` isn't listed as a possible value for the **guestOrExternalUserTypes** property? Please, ensure that the **guestOrExternalUserTypes** property lists all possible values for the **conditionalAccessGuestOrExternalUserTypes** enum.

Same here: 
- api-reference/beta/resources/conditionalaccessguestsorexternalusers.md

**Comment by JarbasHorst** [comment]
Hello @dmytrii-ilchuk, I reviewed your PR and left some comments. Please, take a look when you will have time.

File: api-reference/v1.0/resources/conditionalaccessguestsorexternalusers.md (hunk 2 above)
Comment (dmytrii-ilchuk) [reply api-reference/v1.0/resources/conditionalaccessguestsorexternalusers.md:18]: No reason. Updated.

File: api-reference/v1.0/resources/conditionalaccessenumeratedexternaltenants.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+|:-------------|:------------|:------------|
+| members | String collection | Represents a collection of tenant ids in the scope of Conditional Access for guests and external users policy targeting. |
```
[the remark this answers is not in the document]
Comment (dmytrii-ilchuk) [reply api-reference/v1.0/resources/conditionalaccessenumeratedexternaltenants.md:22]: Updated.

**Comment by dmytrii-ilchuk** [comment]
> Hello @dmytrii-ilchuk, I reviewed your PR and left some comments. Please, take a look when you will have time.
Replied to the comments

**Comment by JarbasHorst** [comment]
Hi @Lauragra, this PR is ready for final review.

**Review by FaithOmbongi — APPROVED** [review]
Hi @dmytrii-ilchuk - one small correction. Please update the author metadata from "sandeo" to **SanDeo-MSFT**. We use the GitHub alias for the author.

Otherwise, approved with suggestions.

Thank you @JarbasHorst !

**Pull request merged**